workers-site/index.js

```diff
@@ -41,6 +44,16 @@ async function handleEvent(event) {
         bypassCache: true,
       }
     }
+
+    if (url.pathname === "/" || url.pathname === "/index.html"){
+      const page = await getAssetFromKV(event, options)
+
+      return new HTMLRewriter()
+        .on('head', new StringInjector("OPERATORS", OPERATORS))
```

Comment:
What does this do? Is this needed?

Reply:
Eh, it isn't used on the page anymore, but someone might want to do something with the operators in the JS in the future. I'm also tempted to leave it as this is OS and it's another good example of what you can do with HTMLRewriter.

Reply:
(It injects a global `OPERATORS` into the client-side JS)